src/servlets/bedtime/helpers.ts

```diff
@@ -45,3 +45,11 @@ export const getCollectionPath = ({ ownerHandle, isAlbum, name, id }: {
   name: string,
   id: number
 }) => `${encodeUrlName(ownerHandle)}/${isAlbum ? 'album' : 'playlist'}/${encodeUrlName(name)}-${id}`
+
+// Get hash from string
+export const getHash = (str: string) => (
```

Comment:
let's add a bigger doc-string style comment on what the purpose of this hash fn is